portafly/src/utils/index.ts

```diff
@@ -19,4 +20,37 @@ const combineReducers = (reducerSlices: Record<string, any>) => (
     prevState
   ))
 
-export { createReducer, combineReducers }
+// # Auth
+// Token is stored in JSON and not string since in the future we will be working with JWT
+const getToken = () => JSON.parse(<string>localStorage.getItem('token'))
+const setToken = (token: string | null) => localStorage.setItem('token', JSON.stringify(token))
```

Comment:
I should not save `"null"` but call `localStorage.removeItem` instead. The input should only be `JWT` only (or at this point, Object I guess)
```suggestion
const setToken = (token: AuthToken) => localStorage.setItem('token', JSON.stringify(token))
```